### PR TITLE
apollo-server example use micro instead of express

### PR DESCRIPTION
**Pull request by captDaylight**
@rauchg @msweeneydev I didn't get this in before #338 was merged, but I dug a little bit and replaced `express` with the `apollo-server-micro` package.

🍾

### CODE REVIEW

**Comment by timothyis** [comment]
This is incredible @captDaylight! Thank you so much 🎉 

I'll just test this a moment and let's get this in 👍

File: apollo/index.js [hunk 1]
```diff
@@ -78,7 +77,4 @@ const server = new ApolloServer({
   playground: true,
 });
 
-const app = express();
-server.applyMiddleware({ app, path: '*' });
-
-module.exports = app;
+module.exports = apolloServer.createHandler({ path: '*' });
```
Comment (timothyis) [inline apollo/index.js]: I tried to run this locally but `apolloServer` is not defined. Did you mean this?
```suggestion
module.exports = ApolloServer.createHandler({ path: '*' });
```

Comment (captDaylight) [reply apollo/index.js]: oh ya 🤭was working too quickly. changing that now

Comment (timothyis) [reply apollo/index.js]: I tried to deploy this and it doesn't work with the path set either, so the following would be the method to use: 
```suggestion
module.exports = server.createHandler();
```

**Comment by timothyis** [comment]
@captDaylight We really appreciate your contributions today! Please don't hesitate to send us more. 

Also, if you'd like to join our public education channel on Slack, where we speak about contributions and early access features, feel free to send me an email at timothy(at)zeit.co and I'll invite you 🙌

**Comment by rauchg** [comment]
Yep @captDaylight that was a really great and productive exchange. Thanks for your time!

**Comment by captDaylight** [comment]
My pleasure y'all :) !

**Pull request merged**